.env

```diff
@@ -1,4 +1,4 @@
 USE_NODE_SE=true
-TON_NETWORK_ADDRESS=http://0.0.0.0:8080
+TON_NETWORK_ADDRESS=http://0.0.0.0:8083
```

Comment:
Looks like this change is not related to commit

Reply:
this file is used only by developers